ostd/src/mm/frame/meta.rs

```diff
@@ -82,7 +82,7 @@ pub(in crate::mm) struct MetaSlot {
     ///  - the implementation can simply cast a `*const MetaSlot`
     ///    to a `*const AnyFrameMeta` for manipulation;
     ///  - if the metadata need special alignment, we can provide
-    ///    at most `PAGE_METADATA_ALIGN` bytes of alignment;
+    ///    at most `FRAME_METADATA_MAX_ALIGN` bytes of alignment;
```

Comment:
We can wrap it in square brackets so that rustdoc and analyzer can navigate to the definition.

```suggestion
   ///    at most [`FRAME_METADATA_MAX_ALIGN`] bytes of alignment;
```